packages/dataDisplay/useSortableTable/src/useSortableTable.tsx

```diff
@@ -23,76 +32,100 @@ const _initalizeConvertedData = <V extends Object,>(data: V[], keys: (keyof V)[]
     return initTableData;
 }
 
+
+    /** sort data depends on convertedData, key, compareFn */
+const _updateSortedData = <V extends object>(
+    convertedData: SortableTableDataType<V>,
+    setSortedData: React.Dispatch<React.SetStateAction<V[]>>,
+    key?: keyof V,
+    compareFn?: (a: any, b: any) => number
+)=>{
+    
+    const revertDataInOrder = <V extends object>(convertedData: SortableTableDataType<V>, order?: number[]) => {
+
+        const keys = [...convertedData.keys()];
+            
+        const sortedIndex = order
+            ?? Array(convertedData.get(keys[0])?.length).fill(null).map((v,index) => index);
```

Comment:
```suggestion
        const sortedIndex = order
            ?? convertedData.get(keys[0])?.map((_v, index) => index) ?? [];
```

`keys[0]의 길이 만큼 array 생성 -> null로 채우기 -> index로 바꾸기`의 로직을 
`keys[0]의 길이만큼 index로 채우기`로 줄일 수 있을 것 같아요

Reply:
좋습니다!